src/drv_psvr/psvr.c

```diff
@@ -190,12 +202,12 @@ static hid_device* open_device_idx(int manufacturer, int product, int iface, int
 	hid_device* ret = NULL;
 
 	while (cur_dev) {
-		LOGI("%04x:%04x %s\n", manufacturer, product, cur_dev->path);
+		LOGD("%04x:%04x %s", manufacturer, product, cur_dev->path);
```

Comment:
Should we have a method of showing debug messages without recompiling, say with an environment variable, before hiding these? They can be useful to request from users who have problems.

Reply:
Hmm good point, that feels like something more generic we should add to OpenHMD as a whole.

Tho right now these are useless and just messes up the stdout of applications using OpenHMD, I think the driver has moved beyond having to care about what devices it opens. :)

Reply:
For PSVR on Linux I agree, I just remembered that these have been useful in the past (#95, I had no idea how the path looks like on macOS before this, for example).

If this is about messing up stdout of applications, it would make much more sense to me to reduce the default verbosity so that LOGI messages aren't printed either, instead of demoting individual messages to LOGD. There are more LOGI messages in the Vive and WMR drivers.

Reply:
Ok changed.